### PR TITLE
feat: Adding CONTRIBUTING.md file

### LINKED ISSUE #23: Adding contributing guidelines
I think it is important to have contributing guidelines so that an open-source contributor know how to contribute properly.

### PR DESCRIPTION
**Pull request by KerinPithawala**
Fixes #23.
Please review the PR

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Steps:
+  
+
+  -  Go to [issues](https://github.com/s-ayush2903/Recipebook/issues) & select one upon which you want to work!
+  
+  -  Fork the [repository](https://github.com/s-ayush2903/Recipebook.git)
+  
+  -  Clone the fork [repo](https://github.com/<your_username>/Recipebook)
+     - git clone https://https://github.com/<your_username>/Recipebook
```
Comment (s-ayush2903) [inline CONTRIBUTING.md]: Wherever we're using commands, use 'em between a pair of backticks(*``*), that'll make 'em look like code, e.g., `this is a sample how command should appear in md files`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+     - git checkout -b <Your-Branch-Name>
+  
+  -  Add code/scripts related to your respective issues.
+     - git add <your-contribution>
+ 
+  -  Add a commit message !
+     - git commit -a -m "<Add a message related to changes you made>"
+
+  -  Push changes
+    - git push origin
```
Comment (s-ayush2903) [inline CONTRIBUTING.md]: Change it to `git push <remote_name> <branch_name>`, this is more specific and precise info for git to tell where to push.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+  
+  -  Push changes
+    - git push -u origin <name_of_your_branch>
```
Comment (s-ayush2903) [inline CONTRIBUTING.md]: We won't need this, after considering the trailing comment of mine

**Comment by s-ayush2903** [comment]
Also, include in description: `Fixes #issue_number`

File: CONTRIBUTING.md (hunk 3 above)
Comment (KerinPithawala) [reply CONTRIBUTING.md]: Got it. Will make the changes as required.

**Comment by KerinPithawala** [comment]
Yes, I will.

File: CONTRIBUTING.md (hunk 2 above)
Comment (KerinPithawala) [reply CONTRIBUTING.md]: Will do.

File: CONTRIBUTING.md (hunk 1 above)
Comment (KerinPithawala) [reply CONTRIBUTING.md]: Thanks for the feedback. I will change the file accordingly.

**Review by s-ayush2903 — APPROVED** [review]

**Comment by s-ayush2903** [comment]
@Kerveyfelix I don't remember if I've given you points for creating the issue, is this correct, I'm not sure. Let me know, if it is so then I'll give total points after merging this.

**Comment by KerinPithawala** [comment]
@s-ayush2903 I don’t have a track of my points for this issue😅. So I guess it’s upto you to decide the final points.

**Comment by s-ayush2903** [comment]
Okay, I know I'm lazy and won't have given points for the issue creation, will do that with this merge. Also, keep track of your points, please.

**Comment by KerinPithawala** [comment]
Sure, I will keep track of my points from now.

**Pull request merged**